ext/snowcrash/src/UriTemplateParser.cc

```diff
@@ -9,207 +9,251 @@
 #include "UriTemplateParser.h"
 #include "RegexMatch.h"
 
-using namespace snowcrash;
+#define URI_REGEX "^(http|https|ftp|file)?(://)?([^/]*)?(.*)$"
 
-static bool HasMismatchedCurlyBrackets(const URITemplate& uriTemplate)
-{
-    int openCount = 0;
-    int closeCount = 0;
-
-    for (unsigned int i = 0; i < uriTemplate.length(); i++) {
-        if (uriTemplate[i] == '{')
-            openCount++;
-        if (uriTemplate[i] == '}')
-            closeCount++;
-    }
-    return openCount != closeCount;
-}
+#include "../../../src/parser/Uritemplate.h"
```

Comment:
If we merge this, we should move snowcrash into the `src/parser` directory ASAP so we can avoid this kind of ugliness.

Reply:
Yes it will be better solution, but in current state how is project splited `ext/snowcrash` vs `src/parser` it is not possible.

There is option add into cmake in snowcrash include paths `../../src` but it also is not good idea. 

So we have two options:
- move `src/paser` into `ext/snowcrash/`
- move snowcrash from `ext` to integral part of `drafter`